### DISCUSSION TITLE
Is Gemini-1.5-Pro-002 supported?

### DISCUSSION THREAD

**Discussion by rajivsinclair** [pieces-app/support#416] [Q&A] [answered]
`Gemini-1.5-Pro-002` 
Google released this `002` version of the Gemini 1.5 Pro model on September 24, 2024: https://developers.googleblog.com/en/updated-gemini-models-reduced-15-pro-pricing-increased-rate-limits-and-more/

**Comment by rosie-at-pieces** [comment, marked as answer]
@rajivsinclair Thanks for reaching out to Pieces for Developers! I'm so glad to hear that you like the Gemini models.

It looks like Google bumped Gemini 1.5 Pro to version 002 on October 8th, so if you have Gemini 1.5 Pro selected in the Pieces copilot it will leverage 002!
